app/tests/test_consumer.py

```diff
@@ -12,10 +12,11 @@
 from sdc.crypto.key_store import KeyStore
 from sdc.crypto.exceptions import InvalidTokenException
 from sdc.rabbit.exceptions import QuarantinableError, RetryableError
+from unittest.mock import patch
 import yaml
 
 from app.main import SeftConsumer, KEY_PURPOSE_CONSUMER
-from app.settings import RM_SDX_GATEWAY_URL
+from app.settings import RM_SDX_GATEWAY_URL  # FTP_FOLDER_BASE
```

Comment:
Don't think this needs to be here:  # FTP_FOLDER_BASE

Reply:
Changed